src/frontend/screens/ObservationEdit/index.tsx

```diff
@@ -67,14 +75,14 @@ export const ObservationEdit: NativeNavigationComponent<'ObservationEdit'> & {
       onPress: handleCameraPress,
     },
   ];
-  // if (preset && preset.fields && preset.fields.length) {
-  //   // Only show the option to add details if preset fields are defined.
-  //   bottomSheetItems.push({
-  //     icon: <DetailsIcon />,
-  //     label: t(m.detailsButton),
-  //     onPress: handleDetailsPress,
-  //   });
-  // }
+  if (preset?.fieldIds && preset.fieldIds.length) {
```

Comment:
nit: think this can be shortened?

```suggestion
  if (preset?.fieldIds?.length) {
```